install_config/persistent_storage/persistent_storage_aws.adoc

```diff
@@ -142,3 +142,12 @@ formatted with the given file system.
 
 This allows using unformatted AWS volumes as persistent volumes, because
 {product-title} formats them before the first use.
+
+=== Maximum number of EBS volumes on a node
+By default {product-title} supports maximum of 39 EBS volumes attachable to one Node. This limit is
+consistent with AWS recommended limit - link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS Volume limits]
+
+{product-title} can however be configured to have higher limit than that by setting environment variable *KUBE_MAX_PD_VOLS*. AWS however
+requires a particular naming scheme(link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html[AWS Device naming]) for attached devices
```

Comment:
"AWS however requires a particular naming scheme(link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html[AWS Device naming]) for attached devices and that device naming scheme only supports a maximum of 52 volumes. So - theoretical upper limit of volumes attachable to a node via {product-title} is 52."

Should be (or some variation on this) : 

"You will need to follow the link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html[AWS Device naming] guidelines for attached devices.  Since these guidelines only support a maximum of 52 volumes, the upper limit of volumes that can be attached to a node via {product-title} is 52."

Reply:
I am bothered with `You` word there. It is not user who needs to follow any guidelines. We the authors/creators of Openshift has to follow that guideline and that guideline limits us to 52 volumes. The link is - just to illustrate the fact that until Amazon makes changes in their API - we can not support more than 52 volumes.